jody/hash.go

```diff
@@ -97,9 +97,22 @@ func AddString64(h uint64, s string) uint64 {
 	}
 
 	if n := (r.Len & 7); n != 0 {
-		m := mask64[n]
-		c := constant & m
-		v := *(*uint64)(p) & m // risk of segfault here?
+		c := constant & mask64[n]
+		var v uint64
```

Comment:
To keeping the local style consistent:
```suggestion
		v := uint64(0)
```